### PR TITLE
Weekly Changelog 2023-09-04 to 2023-09-11

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Additional context
`Automatically generated as a draft. Please copy-edit before merging.`

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -47,6 +47,10 @@ Functionality to drop items off ledges/cliffs onto creatures below
 NPC fleeing behaviour adjusted and slightly improved
 Adds basic vehicle proficiencies for driving and boating, as well as athletic proficiencies which increase muscle engine output.
 Add overheat mechanics to energy guns.
+Add spash screen to Object Creator
```
Comment (Maleclypse) [inline data/changelog.txt:50]:
```suggestion
```

File: data/changelog.txt [hunk 2]
```diff
@@ -47,6 +47,10 @@ Functionality to drop items off ledges/cliffs onto creatures below
...
+Add EOC condition: weapon category
+Sound-triggered traps
+Option to ignore zone contents in usage oriented zones
```
Comment (Maleclypse) [inline data/changelog.txt:53]:
```suggestion
```

File: data/changelog.txt [hunk 3]
```diff
@@ -84,6 +88,8 @@ Adds curved road bends, rotaries and rest stops/laybys
 JSONify Defense Mode
 Adds a new mission to the Exodii
 Add boats to river banks
+Add cordless drills to "power tools" itemgroup and create corded power drills
```
Comment (Maleclypse) [inline data/changelog.txt:91]:
```suggestion
```

File: data/changelog.txt [hunk 4]
```diff
@@ -84,6 +88,8 @@ Adds curved road bends, rotaries and rest stops/laybys
...
+Adds pineapple ingredient for Supreme Pizza
```
Comment (Maleclypse) [inline data/changelog.txt:92]:
```suggestion
```

File: data/changelog.txt [hunk 5]
```diff
@@ -106,6 +112,7 @@ Change description of highlighted regions when editing the overmap
 Detailed information for stats on character creation menu
 Larger and more I18n-friendly safemode UI
 Enable/disable showing several non-player-related messages in the log
+Character creation: small screen UI fixes
```
Comment (Maleclypse) [inline data/changelog.txt:115]:
```suggestion
```

File: data/changelog.txt [hunk 6]
```diff
@@ -147,6 +154,19 @@ Add the Mind Over Matter mod to the CDDA repository
 [Magiclysm] Add more than a dozen spells to magiclysm
 [Magiclysm] Add fantasy species starting option
 [Magiclysm] Add two more playable fantasy species for Magiclysm
+Remove default skills from fantasy species choices
```
Comment (Maleclypse) [inline data/changelog.txt:157]:
```suggestion
```

File: data/changelog.txt [hunk 7]
```diff
@@ -147,6 +154,19 @@ Add the Mind Over Matter mod to the CDDA repository
...
+Add overheating to LV-429
+[DinoMod] use EAT_CARRION
+Add seven druid, earthshaper, and technomancer spells
+Xedra Evolved Border Patrol Office Improvements
```
Comment (Maleclypse) [inline data/changelog.txt:161]:
```suggestion
```

File: data/changelog.txt [hunk 8]
```diff
@@ -147,6 +154,19 @@ Add the Mind Over Matter mod to the CDDA repository
 [Magiclysm] Add more than a dozen spells to magiclysm
 [Magiclysm] Add fantasy species starting option
 [Magiclysm] Add two more playable fantasy species for Magiclysm
+Remove default skills from fantasy species choices
+Add overheating to LV-429
+[DinoMod] use EAT_CARRION
+Add seven druid, earthshaper, and technomancer spells
```
Comment (Maleclypse) [inline data/changelog.txt:160]:
```suggestion
```

File: data/changelog.txt [hunk 9]
```diff
@@ -147,6 +154,19 @@ Add the Mind Over Matter mod to the CDDA repository
 [Magiclysm] Add more than a dozen spells to magiclysm
 [Magiclysm] Add fantasy species starting option
 [Magiclysm] Add two more playable fantasy species for Magiclysm
+Remove default skills from fantasy species choices
+Add overheating to LV-429
```
Comment (Maleclypse) [inline data/changelog.txt:158]:
```suggestion
```

File: data/changelog.txt [hunk 10]
```diff
@@ -147,6 +154,19 @@ Add the Mind Over Matter mod to the CDDA repository
...
+[DinoMod] use EAT_CARRION
+Add seven druid, earthshaper, and technomancer spells
+Xedra Evolved Border Patrol Office Improvements
+JSON-ize faction camp hunting returns
+Add nature spirits to Magiclysm, part 1
```
Comment (Maleclypse) [inline data/changelog.txt:163]:
```suggestion
```

File: data/changelog.txt [hunk 11]
```diff
@@ -147,6 +154,19 @@ Add the Mind Over Matter mod to the CDDA repository
...
+[DinoMod] document lore
+Aftershock: Rebalance energy weapons to use overheat mechanics
+Add warning message before dragon breath
+Add fantasy species ferals
+[DinoMod] dino camp hunting
```
Comment (Maleclypse) [inline data/changelog.txt:168]:
```suggestion
```

File: data/changelog.txt [hunk 12]
```diff
@@ -147,6 +154,19 @@ Add the Mind Over Matter mod to the CDDA repository
...
+Fixing up a single issue within BombasticPerks OLD_GUN perk.
```
Comment (Maleclypse) [inline data/changelog.txt:169]:
```suggestion
```

File: data/changelog.txt [hunk 13]
```diff
@@ -178,6 +198,9 @@ Adds a mutable stream to the mapgen
 Limit the times assassins can try to kill you.
 Increase plastic variety, adjust plastics to be more realistic in terms of protection
 Characters start with basic skills from their previous life
+Most NPCs altruism (personality) scores are average (normal distribution) instead of pure random
+Restore the craft amount for arrows to 10
```
Comment (Maleclypse) [inline data/changelog.txt:202]:
```suggestion
```

File: data/changelog.txt [hunk 14]
```diff
@@ -178,6 +198,9 @@ Adds a mutable stream to the mapgen
...
+Rebalance the price of 7.62x39mm and the gunsmith ammo conversion ratio
```
Comment (Maleclypse) [inline data/changelog.txt:203]:
```suggestion
```

File: data/changelog.txt [hunk 15]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
 Fix appliance power drain display right after plugging in/unplugging device
 Fix items and furniture being deleted from grappling hook usage
 Randomly generated characters are now aged appropriately to their profession
+Mind Over Matter typo fix
```
Comment (Maleclypse) [inline data/changelog.txt:273]:
```suggestion
```

**Review by Maleclypse — COMMENTED** [review]
I'll come back to this later.

File: data/changelog.txt [hunk 16]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
...
+Don't apply the default backgrounds for non custom starts
```
Comment (Maleclypse) [inline data/changelog.txt:274]:
```suggestion
```

File: data/changelog.txt [hunk 17]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
...
+Fix spirit faction relations
```
Comment (Maleclypse) [inline data/changelog.txt:275]:
```suggestion
```

File: data/changelog.txt [hunk 18]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
...
+Prevent segfault when scrolling options
```
Comment (Maleclypse) [inline data/changelog.txt:276]:
```suggestion
```

File: data/changelog.txt [hunk 19]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
...
+Re-scroll list of options when toggling header
```
Comment (Maleclypse) [inline data/changelog.txt:277]:
```suggestion
```

File: data/changelog.txt [hunk 20]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
...
+This resolves incorrect RAW flags in raw_fruit.json
```
Comment (Maleclypse) [inline data/changelog.txt:278]:
```suggestion
```

File: data/changelog.txt [hunk 21]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
...
+Fix signal handler: call to exit() in Object Creator
```
Comment (Maleclypse) [inline data/changelog.txt:279]:
```suggestion
```

File: data/changelog.txt [hunk 22]
```diff
@@ -247,6 +270,14 @@ Make sure overconfident officers reliably drop their guns
...
+Fix Cattail Jelly Mission Turn-in
```
Comment (Maleclypse) [inline data/changelog.txt:280]:
```suggestion
```

File: data/changelog.txt [hunk 23]
```diff
@@ -257,6 +288,8 @@ Fix Nested List lag in crafting menu
 Optimize eoc processing and other fixes to speed up waiting near many npcs
 Optimize pocket overflow function
 Speed up new character screen, particularly when many recipes are known
+Cache results of item search functions
```
Comment (Maleclypse) [inline data/changelog.txt:291]:
```suggestion
```

File: data/changelog.txt [hunk 24]
```diff
@@ -257,6 +288,8 @@ Fix Nested List lag in crafting menu
...
+Optimize loading of pockets with many items inside
```
Comment (Maleclypse) [inline data/changelog.txt:292]:
```suggestion
```

File: data/changelog.txt [hunk 25]
```diff
@@ -47,6 +47,8 @@ Functionality to drop items off ledges/cliffs onto creatures below
 NPC fleeing behaviour adjusted and slightly improved
 Adds basic vehicle proficiencies for driving and boating, as well as athletic proficiencies which increase muscle engine output.
 Add overheat mechanics to energy guns.
+Add EOC condition: weapon category
+Sound-triggered traps
```
Comment (dseguin) [inline data/changelog.txt:51]:
```suggestion
Sound-triggered traps
```

File: data/changelog.txt [hunk 26]
```diff
@@ -147,6 +149,12 @@ Add the Mind Over Matter mod to the CDDA repository
 [Magiclysm] Add more than a dozen spells to magiclysm
 [Magiclysm] Add fantasy species starting option
 [Magiclysm] Add two more playable fantasy species for Magiclysm
+[DinoMod] use EAT_CARRION
+JSON-ize faction camp hunting returns
+[DinoMod] document lore
+Aftershock: Rebalance energy weapons to use overheat mechanics
+Add warning message before dragon breath
+Add fantasy species ferals
```
Comment (dseguin) [inline data/changelog.txt:157]:
```suggestion
JSON-ize faction camp hunting returns
[DinoMod] document lore
[Aftershock] Rebalance energy weapons to use overheat mechanics
[Magiclysm] Add fantasy species ferals
```

File: data/changelog.txt [hunk 27]
```diff
@@ -178,6 +186,7 @@ Adds a mutable stream to the mapgen
 Limit the times assassins can try to kill you.
 Increase plastic variety, adjust plastics to be more realistic in terms of protection
 Characters start with basic skills from their previous life
+Most NPCs altruism (personality) scores are average (normal distribution) instead of pure random
```
Comment (dseguin) [inline data/changelog.txt:189]:
```suggestion
```

**Pull request merged**